src/Microsoft.AspNet.OData.Shared/Builder/ODataConventionModelBuilder.cs

```diff
@@ -694,41 +698,52 @@ private void MapStructuralProperty(StructuralTypeConfiguration type, PropertyInf
 
         // figures out the type of the property (primitive, complex, navigation) and the corresponding edm type if we have seen this type
         // earlier or the user told us about it.
-        private PropertyKind GetPropertyType(PropertyInfo property, out bool isCollection, out IEdmTypeConfiguration mappedType)
+        private PropertyKind GetPropertyType(PropertyInfo property, out bool isCollection, out IEdmTypeConfiguration mappedType, out bool IsValid)
```

Comment:
nit: the property is a valid property, it just shouldn't be mapped.  Maybe rename to something like "Unmapped"?

Reply:
I changed to "mapped".